examples/CSVImporter/CSVImporter.go

```diff
@@ -0,0 +1,261 @@
+package main
+
+import (
+    "fmt"
+    "flag"
+    "encoding/csv"
+    "os"
+    "io"
+    "github.com/TheJumpCloud/jcapi"
+)
+
+
+//
+// This program will process each line (record) of a CSV file as a user
+// import request into JumpCloud.
+//
+// The CSV file must have each line formatted as:
+// 
+// first_name, last_name, USER_NAME, EMAIL, uid, gid, SUDO_FLAG, password, host_name, tag_name, admin1, admin2, ...
+//
+// Values shown in all lowercase are optional, while those in all uppercase
+// are required.
+//
+// For each line of the CSV file, this program will:
+//
+// 1 - Insert the USER_NAME specified as a new JumpCloud user, if the name
+//     does not already exist, otherwise this line will be treated as an
+//     update to an existing user matching USER_NAME.
+//
+// 2 - If the password was specified, the USER_NAME will have that password
```

Comment:
I hope, but am not certain that this is the behavior when the user is added with a password immediately. Did you/could you confirm that?